etc/fluent-bit/values.yaml

```diff
@@ -1,69 +1,35 @@
-# Default values for fluent-bit.
-
-# kind -- DaemonSet or Deployment
-kind: DaemonSet
-
-image:
-  repository: fluent/fluent-bit
-  pullPolicy: Always
-  # tag:
-imagePullSecrets: []
-nameOverride: ""
-fullnameOverride: ""
-
-serviceAccount:
-  create: true
-  annotations: {}
-  name:
-
-rbac:
-  create: true
-
-service:
-  type: ClusterIP
-  port: 2020
-  labels:
-    {}
-  annotations:
-    {}
-    # prometheus.io/path: "/api/v1/metrics/prometheus"
-    # prometheus.io/port: "2020"
-    # prometheus.io/scrape: "true"
-
-serviceMonitor:
-  enabled: false
-  # namespace: monitoring
-  # interval: 10s
-  # scrapeTimeout: 10s
-  # selector:
-  #  prometheus: my-prometheus
-
-prometheusRule:
-  enabled: false
-  # namespace: ""
-  # additionnalLabels: {}
-  # rules:
-  # - alert: NoOutputBytesProcessed
-  #   expr: rate(fluentbit_output_proc_bytes_total[5m]) == 0
-  #   annotations:
-  #     message: |
-  #       Fluent Bit instance {{ $labels.instance }}'s output plugin {{ $labels.name }} has not processed any
-  #       bytes for at least 15 minutes.
-  #     summary: No Output Bytes Processed
-  #   for: 15m
-  #   labels:
-  #     severity: critical
+flush: 5
 
 ## https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/configuration-file
 config:
   service: |
     [SERVICE]
-        Flush 1
         Daemon Off
-        Log_Level info
+        Flush {{ .Values.flush }}
+        Log_Level {{ .Values.logLevel }}
         Parsers_File parsers.conf
         Parsers_File custom_parsers.conf
-        HTTP_Server Off
+        HTTP_Server On
+        HTTP_Listen 0.0.0.0
+        HTTP_Port {{ .Values.metricsPort }}
+        Health_Check On
+
+  ## https://docs.fluentbit.io/manual/pipeline/filters
+  filters: |
+    [FILTER]
+        Name grep
+        match *
+        Exclude log Elasticsearch built-in security features are not enabled
```

Comment:
@blootsvoets could you tell me what this line does?

Reply:
Every time Graylog does a request to Elasticsearch, it generates this log message. Then each time new log messages are added, also this message gets added, and a message about this message gets added. It grows exponentially.

This Filter excludes the message from the fluent-bit log collection avoiding a build-up of log messages.

Reply:
Thanks but is "built-in security features are not enabled" part of the grep as well?